src/functionalTest/resources/features/F-001/F-001.feature

```diff
@@ -0,0 +1,161 @@
+@F-001
+Feature: F-001: Retrieval of metadata from doc-store for given document id
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-001
+  Scenario: Successful retrieval of metadata from doc-store for given document id for caseworker
+    Given a user with [an active profile in CCD]
+    And   a user with [an active caseworker profile in CCD with full permission on specific document field]
```

Comment:
are we using two different users here? it seems like only the caseworker with full permissions is making the calls, in which case we only need to mention one user here. same applies to the other scenarios as well